app/components/zendesk-request-form.js

```diff
@@ -40,7 +42,23 @@ export default Component.extend({
     return buildDescriptionTemplate(this.page);
   }),
 
-  isLoggedIn: reads('auth.signedIn'),
+  isSignedIn: reads('auth.signedIn'),
+  isNotSignedIn: not('isSignedIn'),
+  isPro: reads('features.proVersion'),
+
+  subscriptions: reads('accounts.subscriptions'),
+  activeSubscriptions: filterBy('subscriptions', 'isSubscribed', true),
+  isSubscribed: notEmpty('activeSubscriptions'),
+  isEducation: reads('auth.currentUser.education'),
+
+  trial: reads('auth.currentUser.trial'),
+  trialBuildsRemaining: reads('trial.buildsRemaining'),
+  noTrialYet: empty('trial'),
+
+  isPremium: or('isSubscribed', 'isEducation', 'trialBuildsRemaining', 'noTrialYet'),
```

Comment:
I really like how you juggle with computed macros! 😄

Reply:
Thanks! :smile: